Formula/par.rb

```diff
@@ -15,7 +15,7 @@ class Par < Formula
 
   conflicts_with "rancid", because: "both install `par` binaries"
 
-  # A patch by Jérôme Pouiller that adds support for multibyte
+  # A patch by Jerome Pouiller that adds support for multibyte
```

Comment:
I think we should leave this as-is; M. Pouiller’s first name is *Jérôme*, not *Jerome*.